File: TTTChatRoom/TTTChatRoom/TCRUser.m
//
//  TCRUser.m
//  TTTChatRoom
//
//  Created by Work on 2019/1/18.
//  Copyright © 2019 Work. All rights reserved.
//

#import "TCRUser.h"

@implementation TCRUser
- (instancetype)initWithUid:(int64_t)uid {
    self = [super init];
    if (self) {
        _uid = uid;
    }
    return self;
}
@end

File: ChatRoom-Swift/TTTChatRoom/TCRUser.swift
//
//  TCRUser.swift
//  TTTChatRoom
//
//  Created by Work on 2019/3/13.
//  Copyright © 2019 yanzhen. All rights reserved.
//

import UIKit
import TTTRtcEngineVoiceKit

class TCRUser: NSObject {

    var uid: Int64 = 0
    var role = TTTRtcClientRole.clientRole_Audience
    var mutedSelf = false //是否静音
    var audioLevel: UInt = 0
    init(_ uid: Int64) {
        self.uid = uid
    }
}
